lib/datadog/appsec/contrib/rails/ext.rb

```diff
@@ -5,7 +5,6 @@ module Rails
         # Rack integration constants
         module Ext
           APP = 'rails'.freeze
```

Comment:
I know this is a pattern we've used elsewhere in ddtrace but... it's a really high overhead creating a whole new module to contain `'rails'` (or `'rack'`).

Not for this PR of course, but some food for thought if we still think these add any value.